### ISSUE TITLE
Errors on Windows during provision

### ISSUE THREAD

**Issue by mortona42** [forumone/web-starter#18]
looks like an issue with os-detect.sh

```
==> default: Running provisioner: shell...
    default: Running: C:/Users/Andrew/AppData/Local/Temp/vagrant-shell20140621-1
0904-1nvfflx.sh
: command not foundnt/puppet/shell/os-detect.sh: line 2:
: command not foundnt/puppet/shell/os-detect.sh: line 5:
: command not foundnt/puppet/shell/os-detect.sh: line 11:
==> default: /vagrant/puppet/shell/os-detect.sh: line 19: syntax error near unex
pected token `elif'
==> default: /vagrant/puppet/shell/os-detect.sh: line 19: `    elif [ $(which ls
'_release) ]; then
: command not foundnt/puppet/shell/os-detect.sh: line 2:
: command not foundnt/puppet/shell/os-detect.sh: line 5:
: command not foundnt/puppet/shell/os-detect.sh: line 11:
==> default: /vagrant/puppet/shell/os-detect.sh: line 19: syntax error near unex
pected token `elif'
==> default: /vagrant/puppet/shell/os-detect.sh: line 19: `    elif [ $(which ls
'_release) ]; then
==> default: Installing git
```

**Comment by wwhurley** [comment]
What version of Vagrant and what environment are you running this under? Cygwin, Powershell, Git shell?

**Comment by mortona42** [comment]
Vagrant 1.6.3 running in Git shell.

**Comment by wwhurley** [comment]
Resolved with fix from #43
